atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -780,14 +780,18 @@ private void extractLocalWritesForRow(
                         tableRef,
                         cells,
                         (table, uncached) -> getInternal(
-                                "get", tableRef, cells, immediateKeyValueService, immediateTransactionService));
+                                "get", tableRef, uncached, immediateKeyValueService, immediateTransactionService));
```

Comment:
This is bad! We don't need to get all cells remotely, only the uncached ones!

Reply:
nice catch 🍡